extension/src/popup/components/account/AssetDetail/index.tsx

```diff
@@ -20,7 +20,7 @@ import { useAssetDomain } from "popup/helpers/useAssetDomain";
 import { navigateTo } from "popup/helpers/navigate";
 import { formatTokenAmount } from "popup/helpers/soroban";
 import { getAssetFromCanonical } from "helpers/stellar";
-import { SimpleBarWrapper } from "popup/basics/SimpleBarWrapper";
+// import { SimpleBarWrapper } from "popup/basics/SimpleBarWrapper";
```

Comment:
We don't need to keep the `SimpleBarWrapper` to use JS for content scrolling, right? I kept it in most places where it didn't affect the new layout, but it seems unnecessary for the new responsive layout. Removing all components and styles associated with it is a big update (many files will be affected), so we might handle that in a separate PR.

Reply:
Is your question if we can get rid of the wrapper and just directly use `import SimpleBar from "simplebar-react";`? We could probably do that. 

But we would still need to use simplebar-react to have a JS scrollable div (especially when we're in a smaller screen/view). For context, we used native browser div scrolling for a while, but it was pretty ugly. This created a nicer UI